### DISCUSSION TITLE
What if grid tables could sort your items?

### DISCUSSION THREAD

**Discussion by arnonrdp** [quasarframework/quasar#12159] [Ideas / Proposals] [answered]
Today I was dealing with tables in grid format and at one point I needed them to be sorted, so I sorted the data before it arrived in the table, so far so good.

But it would also be nice if I had a slot in the header that would allow me to change the sort key.

What do you think about it?

**Comment by MilosPaunovic** [comment]
Take a look at the `grid-header` prop of `QTable`, or you could use some of the slots provided to create your own buttons (or other control) for sorting.

**Reply by arnonrdp** [reply]
I will try. Thanks for suggestion.

**Reply by arnonrdp** [reply]
@MilosPaunovic, I followed your suggestion but as per my need (I need to sort out of the table) and I got stuck trying to do that.

Do you mind taking a look at what I've done?

https://codepen.io/arnonrdp/pen/gOGVBqO

**Reply by MilosPaunovic** [reply]
What exactly are you trying to accomplish, to have a select box on the top and to choose a filed to sort by?

**Reply by arnonrdp** [reply]
Exactly! Haha

**Comment by MilosPaunovic** [comment, marked as answer]
[https://codepen.io/milospaunovic/pen/eYeOYWK?editors=1011](https://codepen.io/milospaunovic/pen/eYeOYWK?editors=1011)

**Reply by arnonrdp** [reply]
I really appreciate it, @MilosPaunovic! Didn't know how to use `@update:model-value`.

Thank you so much!
